pkgs/applications/video/frigate/default.nix

```diff
@@ -136,11 +113,12 @@ python.pkgs.buildPythonApplication rec {
     pydantic
     pytz
     pyyaml
+    pyzmq
     requests
     ruamel-yaml
     scipy
     setproctitle
-    tensorflow
+    tensorflow-bin
```

Comment:
This crashes with `illegal instruction` on my older machine because apparently `tensorflow-bin` requires some newer instructions.

In normal `tensorflow`, that is configurable and documented:

https://github.com/NixOS/nixpkgs/blob/18536bf04cd71abd345f9579158841376fdd0c5a/pkgs/development/python-modules/tensorflow/default.nix#L80-L82

* Is using `tensorflow-bin` here necessary?
* Does upstream document anywhere which version it needs?
* Could we change the `tensorflow-bin` derivation so that `-bin` is added to `pname`, so one can easily see which one is in use (CC maintainers @jyp @abbradar)?

Reply:
The source-built package is out of date and barely maintained with known security issues.

Reply:
@mweinelt I see. That seems like a problem for all of `nixpkgs` (currently `frigate` seems to be the only one using `tensorflow-bin`).

Do you know this one?

> Does upstream document anywhere which version it needs?

Reply:
They don't pin a tensorflow version, so I expect anything >=2.0 will work.